_layouts/chapter.html

```diff
@@ -0,0 +1,27 @@
+---
+layout: page
+---
+{{ content }}
+
+{% capture /dev/null %}
+  {% capture _previous_chapter %}[Table of contents](../){% endcapture %}
+  {% for chapter in site.data.scaling.toc %}
+    {% if _chapter_match %}
+      {% capture _next_chapter %}[{{chapter.name}}]({{chapter.permalink}}){% endcapture %}
+      {% break %}
+    {% endif %}
+    {% if chapter.permalink == page.url %}
+      {% assign _chapter_match = true %}
+    {% endif %}
+    {% unless forloop.first %}
+      {% capture _previous_chapter %}[{{chapter.name}}]({{chapter.permalink}}){% endcapture %}
+    {% endunless %}
+  {% endfor %}
+  {% unless _chapter_match %}{% include ERROR72_no_matching_chapter %}{% endunless %}
+  {% unless _next_chapter %}{% capture _next_chapter %}[Table of contents](../){% endcapture %}{% endunless %}
```

Comment:
I think it'd make more sense to print something like "return to table of contents" in the last chapter, rather than "Next Chapter: table of contents"

Reply:
fixed